sympy/integrals/transforms.py

```diff
@@ -1137,6 +1190,607 @@ def sbs(expr):
         aux = _simplifyconds(aux, s, a)
     return _simplify(F.subs(s, s_), simplify), sbs(a), _canonical(sbs(aux))
 
+def _laplace_deep_collect(f, t):
+    func = f.func
+    args = [*f.args]
+    if len(f.args) == 0:
+        return f
+    else:
+        for k in range(len(args)):
+            args[k] = _laplace_deep_collect(args[k], t)
+        if func.is_Add:
+            return func(*args).collect(t)
+        else:
+            return func(*args)
+
+def _laplace_build_rules(t, s):
```

Comment:
This function should have some sort of comment or docstring that explains what it returns and what the purpose is.

Reply:
Will do.